e2e-tests/web/capabilities.ts

```diff
@@ -36,16 +38,11 @@ export const browserstackCapabilities = {
     browserName: 'Safari',
     browserVersion: '14.0'
   })
-}
+} as Record<typeof browsers[number], Capabilities>
```

Comment:
If you remove `edge` from `browsers` you can just add a type annotation instead of having to cast it. But probably it is intended, right?